pandas/tseries/index.py

```diff
@@ -694,6 +694,21 @@ def astype(self, dtype):
         else:  # pragma: no cover
             raise ValueError('Cannot cast DatetimeIndex to dtype %s' % dtype)
 
+    def strftime(self, date_format):
```

Comment:
I thought about something like that too, good idea for code reuse! Will update shortly